pilot/pkg/networking/util/util.go

```diff
@@ -71,6 +71,14 @@ const (
 	IstioMetadataKey = "istio"
 	// The range of LoadBalancingWeight is [1, 128]
 	maxLoadBalancingWeight = 128
+
+	// AcceptMTLSMetadataLabel having "acceptMTLS":"true" key/value pair set on EDS endpoints
+	// configures Envoy to use the "enableMTLS" socket configuration configured at cluster level
+	AcceptMTLSMetadataLabel = "acceptMTLS"
```

Comment:
I thought we use `security.istio.io/mtlsReady: true/false`? no?

Reply:
per discussions regarding envoy memory footprint size concerns, using shortname `mtlsReady`